tokens-generated/css/core.css

```diff
@@ -129,5 +129,34 @@
   --wds-space-800: 4rem;
   --wds-space-900: 4.5rem;
   --wds-space-1200: 6rem;
-  --wds-space-base: 0.5rem;
+  --wds-space-050: 0.25rem;
+  --wds-layer-1: 1;
```

Comment:
@dana-saur, from the tokens.json file, I can tell **layer** means z-index values. Do you think we could switch these to use 'z-index' instead of 'layer'? It might make things a bit more straightforward.